packages/block-editor/src/components/block-list/index.js

```diff
@@ -23,6 +23,22 @@ import useBlockDropZone from '../use-block-drop-zone';
  */
 const BLOCK_ANIMATION_THRESHOLD = 200;
 
+function mergeRefs( ...refs ) {
```

Comment:
Looks like we cannot use the `react-merge-refs` utility here since it returns a function ref instead of an object ref.

Reply:
We can do PR upstream as well.

Reply:
@youknowriad What's the best package for this? `compose`?

Reply:
compose seems like a good one yes.